### DISCUSSION TITLE
Triggering a relay

### DISCUSSION THREAD

**Discussion by SteveJack34** [CJNE/ha-myenergi#550] [Q&A]
I'm trying to trigger a relay on my Eddi. The automation triggers and gets to the delay after the first action, but the relay isn't triggered. Is there something I need to set in the eddi? This is my YAML.
`alias: Daily Hot Water
description: Daily Hot Water Heating
trigger:
  - platform: state
    entity_id:
      - binary_sensor.hot_water_timer
    from: "off"
    to: "on"
condition:
  - condition: state
    entity_id: switch.wiser_away_mode
    state: "off"
    enabled: true
action:
  - service: myenergi.myenergi_eddi_boost
    target:
      device_id:
        - XXXXXX
    data:
      target: Relay 1
      time: 60
  - wait_for_trigger:
      - platform: template
        value_template: >-
          {{ states('sensor.myenergi_hot_water_temp_tank_1') | float > 65 and
          states('sensor.myenergi_hot_water_temp_tank_2') | float > 65 }}
        for:
          hours: 0
          minutes: 0
          seconds: 30
    timeout:
      hours: 1
      minutes: 0
      seconds: 0
      milliseconds: 0
  - service: myenergi.myenergi_eddi_boost
    data:
      target: Relay 1
      time: 0
    target:
      device_id: XXXXXX
mode: single
`
Steve

**Comment by G6EJD** [comment]
Although this is set in the integration are you sure it is a viable option.
As far as I understand the relays are intramurally controlled by the Eddi based on measured parameters, such as eg excess solar after tank-1 has finished or whatever.
I have never seen anyone control the relays via the API and I think it’s not possible.

**Reply by SteveJack34** [reply]
If its not an option and doesn't work then it needs removing.

**Comment by SteveJack34** [comment]
After a bit of playing it does work sort of. You need to set Relay 1 to Boost. The issue no is that the Eddi boost timer may override HA.

**Reply by G6EJD** [reply]
Yes it does but that’s by design, it’s not an independent relay control

**Comment by G6EJD** [comment]
I can see why Christian included it in the logic, because it may be implemented at some time in the future. But it does t work yet. I think you should close this issue unless you can categorically test as not working. 

Don’t forget this is a user based reverse engineered rendition of the API, so ME may be able to do this, but they’ve never released the details of method. The Eddi relay board was in my opinion never designed to interact with the API other than report its status.
